pkgs/development/compilers/ghc/with-packages.nix

```diff
@@ -44,6 +47,7 @@ buildEnv {
       rm -f $out/bin/$prg
       makeWrapper ${ghc}/bin/$prg $out/bin/$prg         \
         --add-flags '"-B$NIX_GHC_LIBDIR"'               \
+        ${optionalString exposeGHC "--add-flags '-package ghc'"} \
```

Comment:
@jwiegley @gridaphobe Any objection to me closing this PR, reverting this change in joelteon/master, and leaving it to @gridaphobe to reopen as a PR unassociated with the darwin changes?